### PR TITLE
Added command for migrations.

### LINKED ISSUE #77: add migrations subcommand
As discussed, we are currently envisioning a call from the manage-service to the backend to initiate pending migrations.
@jsangmeister will implement an HTTP-route the backend to wrap the internal `migrate.py` script.
This will be somewhat of the form `/system/migrations`.
The body of a request to that should be a json object of the form `{ "cmd": "migrate" | "finalize" | "stats" | "reset" | "clear-collectionfield-tables", "flags": ["verbose"] }` (flags can also be an empty list)
corresponding to the subcommands of `migrate.py` (see https://github.com/OpenSlides/openslides-backend/blob/master/migrations/migrate.py).
The `openslides` command should in turn provide these to the user

So as an example a user would want to check if migrations are needed, so they could:
`./openslides migrations stats --verbose`
which sends an grpc call to manage-server -> manage-server sends HTTP request with body `{ "cmd": "stats", "flags": "verbose" }` to `backend/system/migrations` -> HTTP response will include `manage.py`'s answer which can be relayed back to the user.

Similarly the user could call
`./openslides migrations migrate`
`./openslides migrations finalize`
`./openslides migrations reset`
`./openslides migrations clear-collectionfield-tables`

### PR DESCRIPTION
**Pull request by normanjaeckel**
Fixed #77

### CODE REVIEW

File: pkg/migrations/migrations.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	"github.com/OpenSlides/openslides-manage-service/pkg/connection"
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	// MigrationsHelp contains the short help text for the command.
+	MigrationsHelp = "Wrapper to the OpenSlides backend migration tool for allying migrations to the datastore."
```
Comment (peb-adr) [inline pkg/migrations/migrations.go:18]: typo allying

File: pkg/action/action.go [hunk 2]
```diff
@@ -88,5 +104,56 @@ func (c *Conn) Single(ctx context.Context, name string, data json.RawMessage) (j
...
+	}
+	req, err := http.NewRequestWithContext(ctx, "POST", addr, bytes.NewReader(encodedBody))
+	if err != nil {
+		return nil, fmt.Errorf("creating request to backend action service: %w", err)
+	}
+	creds := shared.BasicAuth{
+		Password: c.internalAuthPassword,
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set(shared.AuthHeader, creds.EncPassword())
```
Comment (peb-adr) [inline pkg/action/action.go:133]: I think all of these are basically identical to `Single()`.
You might want to put them into a new method.

**Review by peb-adr — CHANGES_REQUESTED** [review]

File: pkg/migrations/migrations.go (hunk 1 above)
Comment (normanjaeckel) [reply pkg/migrations/migrations.go:18]: I do not understand. Should I write `applying`

File: pkg/action/action.go (hunk 2 above)
Comment (normanjaeckel) [reply pkg/action/action.go:133]: Yes I know. This kind of refactoring has to be done in an extra step.

File: pkg/migrations/migrations.go (hunk 1 above)
Comment (peb-adr) [reply pkg/migrations/migrations.go:18]: Yes, i was under the impression that's what you wanted to write. And I also thin It'd make sense

**Comment by normanjaeckel** [comment]
@peb-adr Can I merge this?

**Review by peb-adr — APPROVED** [review]
As you, did the postgres user thing within this too, I added a small hint which I hope will prevent some problems for user.
Other wise looks good for merging.

**Pull request merged**